### DISCUSSION TITLE
v2/zh/guide/blog/blogger

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#341] [Announcements]
# v2/zh/guide/blog/blogger

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/guide/blog/blogger.html

**Comment by wukongnotnull** [comment]
请问 B站 飞书 社交媒体名称是什么

**Reply by Mister-Hope** [reply]
> 如果社交媒体已在下方列表中，你可以直接设置 社交媒体名称: 社交媒体地址。
>
> 否则，你应该传入一个元组 社交媒体名称: [社交媒体地址, 社交媒体 SVG 图标字符串或路径]

没有的你想叫啥叫啥，另外这两个媒体已经在最新的commit中添加。

**Comment by Qh-digit** [comment]
"Wechat": 社交媒体地址是什么形式的？如何获取？

**Reply by Mister-Hope** [reply]
你可以自己做一个页面并设置此页面链接，也可以直接填写你二维码图片地址。

**Comment by Qh-digit** [comment]
明白了，多谢














在 2023-02-05 00:45:22，"Mr.Hope" ***@***.***> 写道：

你可以自己做一个页面并设置此页面链接，也可以直接填写你二维码图片地址。

**Comment by yn-zxj** [comment]
例子中是medias 写成了media
